src/jit/lclvars.cpp

```diff
@@ -5925,7 +5962,7 @@ void   Compiler::lvaDumpEntry(unsigned lclNum, FrameLayoutState curState, size_t
         }
     }
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef is not needed since lvIsHfaRegArg direct field reference is replaced with lvIsHfaRegArg() accessor.